templates/dashboard.jinja.html

```diff
@@ -361,6 +390,25 @@ <h1>
   {% endfor -%}{# svg in svgs #}
   </div><!-- class="graphs" -->
 
+  <h2 class="downloads-header">Downloads</h2>
+  <div class="downloads">
+    <ul>
+      <li>
+        <p>
+          Raw data for this run: <code><a
+          href="run.json">run.json</a></code>
+        </p>
+      </li>
+      {% if litani_zip_path is not none %}
+      <li>
+        <p>HTML report: <code><a
+          href={{litani_zip_path}} download>{{litani_zip_path}}</a></code>
```

Comment:
Jinja style is to have whitespace within expression and statements: `{{ litani_zip_path }}` rather than `{{litani_zip_path}}`. I prefer this, it makes it standout from the text

Reply:
`href={{litani_zip_path}} download>` looks incorrect, do you mean `href={{litani_zip_path}}>download`?

Reply:
And actually to be good HTML it should really be `href="{{ litani_zip_path }}">download`

Reply:
Download is actually an [anchor tag attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ahttps://developer.mozilla.org/en-US/docs/Web/HTML/Element/a). Let me know if that is not what we want here.

Reply:
Oh, very cool! Thanks! But I still think there should be quotes around the href `href="{{ litani_zip_path }}"`